cura/OAuth2/AuthorizationRequestHandler.py

```diff
@@ -127,6 +127,7 @@ def _handleNotFound() -> ResponseData:
     def _sendHeaders(self, status: "ResponseStatus", content_type: str, redirect_uri: str = None) -> None:
         self.send_response(status.code, status.message)
         self.send_header("Content-type", content_type)
+        self.send_header("Strict-Transport-Security", "max-age=900")
```

Comment:
Do we need to have `includeSubdomains` as well? We might not need to, since it's a single endpoint.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Strict-Transport-Security

Reply:
Yeah the scope is extremely limited, if I'm correct we don't even use domains in this case because the callback usually goes directly to localhost, so let's keep it simple